diff -ru ilmbase-2.0.1-orig/CMakeLists.txt ilmbase-2.0.1/CMakeLists.txt
--- ilmbase-2.0.1-orig/CMakeLists.txt	2013-06-19 05:51:38.000000000 +1000
+++ ilmbase-2.0.1/CMakeLists.txt	2013-11-20 00:06:40.559084314 +1000
@@ -112,9 +112,9 @@
 
 # Tests
 
-ADD_SUBDIRECTORY ( HalfTest )
-ADD_SUBDIRECTORY ( IexTest )
-ADD_SUBDIRECTORY ( ImathTest )
+#ADD_SUBDIRECTORY ( HalfTest )
+#ADD_SUBDIRECTORY ( IexTest )
+#ADD_SUBDIRECTORY ( ImathTest )
 
 # Installation
 
diff -ru ilmbase-2.0.1-orig/Half/CMakeLists.txt ilmbase-2.0.1/Half/CMakeLists.txt
--- ilmbase-2.0.1-orig/Half/CMakeLists.txt	2013-06-19 05:51:38.000000000 +1000
+++ ilmbase-2.0.1/Half/CMakeLists.txt	2013-11-18 23:54:30.444437516 +1000
@@ -47,6 +47,7 @@
 
 INSTALL ( FILES 
   half.h
+  halfExport.h
   halfFunction.h
   halfLimits.h
   DESTINATION
