diff -ur lastools-orig/LASzip/src/mydefs.hpp lastools/LASzip/src/mydefs.hpp
--- lastools-orig/LASzip/src/mydefs.hpp	2013-11-21 00:21:48.676523303 +1000
+++ lastools/LASzip/src/mydefs.hpp	2013-11-21 00:22:51.348523296 +1000
@@ -53,7 +53,7 @@
 typedef float              F32;
 typedef double             F64;
 
-#if defined(_MSC_VER) && (_MSC_VER < 1300) || defined (__MINGW32__)
+#if defined(_MSC_VER) || defined (__MINGW32__)
 typedef int                BOOL;
 #else
 typedef bool               BOOL;
Binary files lastools-orig/LASzip/src/.mydefs.hpp.swp and lastools/LASzip/src/.mydefs.hpp.swp differ
